src/docs/product/performance/resources.md

```diff
@@ -0,0 +1,120 @@
+---
+title: Resources
+sidebar_order: 50
+description: "Learn more about browser resource performance monitoring, which allows you to see debug the performance of loading JavaScript and CSS on your frontend."
+---
+
+If you have performance monitoring enabled for your frontend, you can see how your browser resources are performing in Sentry.
+
+Starting with the [**Resources** page](#resources-page), you get a high-level overview of how your resources are doing. From there, you can drill into a specific query's [**Resource Summary** page](#resource-summary-page) and then investigate sample events from the [Sample List](#sample-list) to better understand the context of its performance in a specific page.
+
+<!-- TODO add gif -->
+
+## Prerequisites and Limitations
+
+The resource pages are only available for frontend JavaScript projects with performance monitoring enabled. You need to enable automatic instrumentation via the `BrowserTracing` integration for your frontend project to see resource performance data.
+
+Automatic Instrumentation for frontend JavaScript is supported for the following platforms:
```

Comment:
```suggestion
The resource pages are only available for frontend JavaScript projects with performance monitoring enabled. 

For the best experience, we recommend enabling automatic instrumentation via the `BrowserTracing` integration for your frontend project to see resource performance data. This is supported for the following JavaScript platforms:
```

Original wording implied that resources is only available with automatic instrumentation, but from the next section it sounds like manual also works, so reworded this a bit.

Reply:
good point!